tests/Assetic/Test/Asset/HttpAssetTest.php

```diff
@@ -15,14 +15,14 @@
 
 class HttpAssetTest extends \PHPUnit_Framework_TestCase
 {
-    const JQUERY = 'http://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js';
```

Comment:
What is the relation with `/usr/bin/env: node: No such file or directory` error?